app/controllers/application_templates_controller.rb

```diff
@@ -27,7 +27,7 @@ def toggle_active
     else show_message :inactive_application,
                       default: 'The application is now inactive.'
     end
-    redirect_to :back
+    redirect_back fallback_location: main_dashboard_path
```

Comment:
My brief Googling says that there isn't a built-in way, no.  You could write a method that did that, I guess.

I think that the thinking was that it'd be pretty rare that the fallback "back" location would be the same site-wide.